pyproject.toml

```diff
@@ -42,8 +42,8 @@ build-backend = "setuptools.build_meta"
 skip = ["pp*", "*-musllinux*"]
 test-extras = "test"
 test-command = [
-    "PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/runner.py",
-    "PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/test_memleaks.py"
+    "env PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 PSUTIL_SCRIPTS_DIR={project}/scripts python {project}/psutil/tests/runner.py",
+    "env PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 PSUTIL_SCRIPTS_DIR={project}/scripts python {project}/psutil/tests/test_memleaks.py"
```

Comment:
what is "env" here?

Reply:
Not a pre-requisite for merging this PR, just mentioning it for the future, but we should apply the same fix to the Makefile:
https://github.com/giampaolo/psutil/blob/1aa9bc838a7dbc96c40d9ffef712308205caa7e8/Makefile#L48

Reply:
env is https://man7.org/linux/man-pages/man1/env.1.html